docs/developers/design/001_linear_spinwave_theory.md

```diff
@@ -337,3 +326,60 @@ Instead SpinW uses one of two algorithms:
   if the $`h(\mathbf{q})`$ matrix is not only Hermitian but also positive definite.
 * That of [White et al.](https://doi.org/10.1103/PhysRev.139.A450) for a general $`h(\mathbf{q})`$ matrix.
   Note that in this case SpinW can return (unphysical) imaginary magnon energies.
+
+## The spin-spin correlation function
+
+Once the Hamiltonian matrix has been diagonalised, the eigenvalues and eigenvectors can be used to compute the spin-spin correlation function
+which is directly related to the inelastic neutron scattering cross-section.
+
+Note that this calculation assumes that the eigenvalues (and their associated eigenvectors) are in decreasing order.
+
+For a matrix of eigenvectors $`U`$ and a $`2N \times 2N`$ diagonal matrix $`E = gL`$ where $`L`$ is the diagonal matrix of eigenvalues (the magnon energies),
```

Comment:
I don't think actually define `g` in the above discussion of thr Hamiltonian - it would be good yo define it here.